src/main/target/ALIENFLIGHTF3/target.c

```diff
@@ -64,37 +63,28 @@ const uint16_t airPWM[] = {
 
 
 const timerHardware_t timerHardware[USABLE_TIMER_CHANNEL_COUNT] = {
-    //
     // 6 x 3 pin headers
-    //
+    { TIM15, IO_TAG(PB15), TIM_Channel_2, TIM1_BRK_TIM15_IRQn,     1, IOCFG_AF_PP, GPIO_AF_1 }, // PWM1  - PB15 - TIM1_CH3N, TIM15_CH1N, *TIM15_CH2
```

Comment:
The aim is to have the same target.c files, the same drivers and, very similar target.h files.